src/hypofuzz/interface.py

```diff
@@ -95,16 +95,23 @@ def pytest_collection_finish(self, session: pytest.Session) -> None:
                     flush=True,
                 )
                 continue
-
             # Wrap it up in a FuzzTarget and we're done!
             try:
-                # Skip state-machine classes, since they're not
-                if isinstance(item.obj, RuleBasedStateMachine.TestCase):
-                    target = partial(run_state_machine_as_test, item.obj)
+                if hasattr(item.obj, "_hypothesis_state_machine_class"):
```

Comment:
Rather than attaching a magic attribute - which could be confused by mocks or whatever - let's define a subclass of `TestCase` in `hypothesis.stateful` and then [inherit from it here](https://github.com/HypothesisWorks/hypothesis/blob/eaa0852472ff4830782bfabb9043b914ef989332/hypothesis-python/src/hypothesis/stateful.py#L428-L443), so we have a principled way to identify it on the Hypofuzz side.